File: week5/thief.py
n, t = input().split()
##t = int(t)

##stl=list(map(int,input("").split()))
stl = input().split()

##stl.sort(reverse = True)
##print(stl)

sum = 0
for i in range(int(t)):
    sum += int(stl[-1-i])
##    print(stl[-1-i])

print(sum)

File: week5/thief.cpp
#include<iostream>
#include <algorithm> 
using namespace std;
int main() 
{ 
	int n,t,sum=0;
	cin >> n>>t;
	int arr[n];
	for(int i=0;i<n;i++){
		cin>>arr[i];
	} 
	sort(arr,arr+n);
	
    for (int i = 0; i < t; ++i) 
    	sum+=arr[n-i-1];
//        cout << arr[n-i-1] << " "; 
	cout<<sum;
    return 0; 
} 
